<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Trigger_Configuration</class>
 <widget class="QDialog" name="Trigger_Configuration">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Trigger Configuration</string>
  </property>
  <widget class="QPushButton" name="pushButton_TriggConfig_OK">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>610</y>
     <width>92</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_TriggConfig_Cancel">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>610</y>
     <width>92</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_SoftwareTriggerMode">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>70</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>DISABLED</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>TRIGGER OUTPUT</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ACQUISITION TRIGGER</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ACQUISITION &amp; TRIGGER OUTPUT</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>69</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Software Trigger Mode</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_ExternalTriggerInputMode">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>DISABLED</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>TRIGGER OUTPUT</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ACQUISITION TRIGGER</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ACQUISITION &amp; TRIGGER OUTPUT</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>External Trigger Input Mode</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_ChannelSelfTrigger">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>490</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>DISABLED</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>TRIGGER OUTPUT</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ACQUISITION TRIGGER</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ACQUISITION &amp; TRIGGER OUTPUT</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>490</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Channel Self-Trigger</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_TriggConfig_Load">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>610</y>
     <width>131</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>Load settings</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_TriggConfig_Save">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>610</y>
     <width>131</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>Save settings</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_ChannelTriggerThreshold">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>440</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>2000</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Channel Trigger Threshold</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>I/O Level</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_IOLevel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>150</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>NIM</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>TTL</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Trigger Polarity</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_TriggerPolarity">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>390</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Rising Edge</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Falling Edge</string>
    </property>
   </item>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_ChannelTriggerThreshold">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>440</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="maximum">
    <double>2000.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Run Synchronization Mode</string>
   </property>
  </widget>
  <widget class="QComboBox" name="RunSynchronizationMode">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>190</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>DISABLED</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>TRG-OUT / TRG-IN DAISY CHAIN</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>TRG-OUT / S-IN CONTROLLED</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>S-IN CONTROLLED</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_Channels">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>320</y>
     <width>121</width>
     <height>36</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Channel 0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Channel 1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Channel 2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Channel 3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Channel 4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Channel 5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Channel 6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Channel 7</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>General settings</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>270</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Settings for selected channel</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_EveryChannel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>241</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Same settings for every channel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
